exercises/prime-factors/prime_factors_test.exs

```diff
@@ -62,4 +62,14 @@ defmodule PrimeFactorsTest do
   test "93819012551" do
     assert PrimeFactors.factors_for(93819012551) == [11, 9539, 894119]
   end
+
+  @tag :pending
+  # @tag timeout: 2000
+  #
+  # The above tag will set the below test to fail unless it completes in under
```

Comment:
Sorry to nitpick.

Perhaps `The 'timeout' tag above will set the test below to`?